packages/modules-sdk/src/medusa-module.ts

```diff
@@ -48,6 +49,23 @@ type ModuleAlias = {
   main?: boolean
 }
 
+export type ModuleBootstrapOptions = {
+  moduleKey: string
+  defaultPath: string
+  declaration?: InternalModuleDeclaration | ExternalModuleDeclaration
+  moduleExports?: ModuleExports
+  globalContainer?: MedusaContainer
```

Comment:
whats a globalContainer? How is this different from a shared container?

Reply:
it is just the naming, but there is no shared here but it could be called shared

Reply:
but otherwise, medusa app receive a container (shared from outside) and then declare a globalContainer from now on that is a child of the shared one, meaning that everything added to the child should not be present in the shared. is that answer helping? :joy:

Reply:
I ve renamed for clarity

Reply:
Yes, makes sense. Just wanted to see why this was differently named here. Thanks!